DDCAD/src/ASSIMPWriter.cpp

```diff
@@ -157,13 +158,15 @@ namespace  {
             ++nskip;
             continue;
           }
+#if ROOT_VERSION_CODE < ROOT_VERSION(6,31,1)
           bool degenerated = true;
```

Comment:
I couldn't figure out yet how this can be done now.

Reply:
@MarkusFrankATcernch Any objection to merging this now, and figuring this part out later?

Reply:
Implemented the function now